PyPDF2/generic.py

```diff
@@ -48,7 +48,7 @@
 
 ObjectPrefix = b_('/<[tf(n%')
 NumberSigns = b_('+-')
-IndirectPattern = re.compile(b_(r"[+-]?(\d+)\s+(\d+)\s+R[^a-zA-Z]"))
+IndirectPattern = re.compile(b_(r"(\d+)\s+(\d+)\s+R[^a-zA-Z]"))
```

Comment:
What does it mean that now +/- are no longer in the pattern? Was it a bug that it was there before?